opencti-platform/opencti-front/src/private/components/settings/Root.jsx

```diff
@@ -215,6 +217,15 @@ const Root = () => {
             path="/dashboard/settings/customization/rules"
             component={Rules}
           />
+          <BoundaryRoute
+            exact
+            path="/dashboard/settings/customization/decay"
+            component={Decay}
+          />
+          <BoundaryRoute
+            path="/dashboard/settings/customization/decay/:decayRuleId"
+            component={RootDecay}
+          />
```

Comment:
In my opinion here we should have only one route `/dashboard/settings/customization/decay` that points to `decay/Root.tsx` and in `decay/Root.tsx` define 2 relative paths `""` and `":decayId"`